pages/components/Badge/badge.html

```diff
@@ -0,0 +1,182 @@
+
+            <h2 class="sub-comp-header">Avatars with Badge</h2>
+
+            <div class="comp-disp-container disp-flex">
+                <figure class="avatar avatar-xl badge disp-inline">
+                    <img
+                        src="https://picturepan2.github.io/spectre/img/avatar-2.png"
+                        alt="avatar demo"
+                        height="4em"
+                    />
+                    <div class="badge-count">99</div>
+                </figure>
+                <figure class="avatar avatar-lg badge disp-inline">
+                    <img
+                        src="https://picturepan2.github.io/spectre/img/avatar-3.png"
+                        alt="avatar demo"
+                        height="3em"
+                    />
+                    <div class="badge-count"></div>
+                </figure>
+                <figure class="avatar avatar-std badge disp-inline">
+                    <img
+                        src="https://picturepan2.github.io/spectre/img/avatar-1.png"
+                        alt="avatar demo"
+                        height="2em"
+                    />
+                    <div class="badge-count">8</div>
+                </figure>
+            </div>
+
+            <p class="comp-desc">
+                The badge component can be used in two ways with and without the
+                count.
+            </p>
+
+            <pre class="comp-code-container"><code class="html">
+    &lt;figure class="avatar avatar-xl badge">
+        &lt;img src="..." alt=""/>
+        &lt;div class="badge-count">8&lt;/div>
+    &lt;/figure>
+    
+    &lt;figure class="avatar avatar-std badge">
+        &lt;img src="..." alt=""/>
+        &lt;div class="badge-count">8&lt;/div>
+    &lt;/figure>
+            </code></pre>
+
+            <h2 class="sub-comp-header">Icons with Badge</h2>
+
+            <div class="comp-disp-container disp-flex">
+                <figure class="badge">
+                    <img src="/images/cart.svg" alt="" />
+                    <div class="badge-count">8</div>
+                </figure>
+                <figure class="badge">
+                    <img src="/images/mail.svg" alt="" />
+                    <div class="badge-count">8</div>
+                </figure>
+            </div>
+            <p class="comp-desc">
+                The badge component can be used with and without the badge count
+            </p>
+
+            <pre class="comp-code-container"><code class="html">
+    &lt;figure class="badge">
+        &lt;img src="..." alt="" />
+        &lt;div class="badge-count">8&lt;/div>
+    &lt;/figure>
+
+    &lt;figure class="badge">
+        &lt;img src="..." alt="" />
+        &lt;div class="badge-count">8&lt;/div>
+    &lt;/figure>
+            </code></pre>
+
+            <footer class="page-footer">
+                <a
+                    href="/pages/components/Alert/alert.html"
+                    class="footer-button"
```

Comment:
```suggestion
            <footer class="page-footer">
                <a
                    href="/pages/components/Alert/alert.html"
                    class="button-footer"
```
try to be consistent with namings